bindings/gumjs/gumdukcore.c

```diff
@@ -2742,7 +2742,9 @@ gum_duk_native_function_invoke (GumDukNativeFunction * self,
 
     if (gum_exceptor_try (core->exceptor, &exceptor_scope))
     {
+      gum_interceptor_unignore_current_thread (core->interceptor);
       ffi_call (&self->cif, implementation, rvalue, avalue);
+      gum_interceptor_ignore_current_thread (core->interceptor);
```

Comment:
- The `ignore` also needs to happen when `gum_exceptor_try` returns `FALSE` because of an unhandled exception resulting in us `setjmp`ing back to the call.
- `core->interceptor` is not the Interceptor, it's DukInterceptor, i.e. the JS module, so this needs an extra de-reference to get the actual `Interceptor`. Would suggest a variable for this.